src/WarcraftLegacies.Source/Factions/Bilgewater.cs

```diff
@@ -23,7 +23,7 @@ public sealed class Bilgewater : Faction
     {
       _allLegendSetup = allLegendSetup;
       StartingGold = 200;
-      StartingLumber = 700;
+      StartingLumber = 00;
```

Comment:
You're missing tonnes of factions here: 
![image](https://github.com/AzerothWarsLR/WarcraftLegacies/assets/8995690/958b36f7-9ca6-4bea-a6be-ef029c2066f5)